JS-Algorithms/challenges.js

```diff
@@ -87,15 +87,15 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  let sum = 0n;
+  let sum = BigInt(0);
 
-  for (let i = 1; i <= number; i++){
-    let pow = BigInt(i) ** BigInt(i);
-    sum += pow;
+  for (let i = BigInt(1); i <= number; i++){
+   sum += i ** i;
```

Comment:
just as a question should we cast the index here? 
sum+= BigInt(i) ** BigInt(i)

Reply:
That is correct